tests/Converter/Validator/Xsd2ValidatorTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+namespace GoetasWebservices\Xsd\XsdToPhp\Tests\Converter\Validator;
+
+class Xsd2ValidatorTest extends Xsd2ValidatorBase
+{
+
+    /**
+     * @dataProvider getRestrictionsValidations
+     */
+    public function testSimpleTypeWithValidations($xsRestrictions, $ymlValidations)
+    {
+        $xml = "
+             <xs:schema targetNamespace=\"http://www.example.com\" xmlns:xs=\"http://www.w3.org/2001/XMLSchema\">
+                <xs:element name=\"element-one\">
+                    <xs:simpleType>
+                         <xs:restriction base=\"xs:string\">
+                            {$xsRestrictions}
+                         </xs:restriction>
+                    </xs:simpleType>
+                </xs:element>
+               </xs:schema>
+            ";
+
+        $classes = $this->getClasses($xml);
+        
+        $this->assertCount(1, $classes);
+
+        $this->assertEquals(
+            [
+                'Example\\ElementOne' => [
+                    'properties' => [
+                        '__value' => $ymlValidations
+                    ]
+                ]
+            ], $classes['Example\ElementOne']);
+    }
+    
+    /**
+     * @dataProvider getRestrictionsValidations
+     */
+    public function testComplexTypeWithValidations($xsRestrictions, $ymlValidations)
+    {
+        $content = '
+             <xs:schema targetNamespace="http://www.example.com" xmlns:xs="http://www.w3.org/2001/XMLSchema"  xmlns:ex="http://www.example.com">
+                <xs:complexType name="complexType-1">
+                    <xs:sequence>
+                        <xs:element name="column1">
```

Comment:
this test is good, I will add another test, having `<xs:element name="column1" maxOccours="unbounded">`